jboss/container/wildfly/galleon/build-feature-pack/configure.sh

```diff
@@ -41,7 +41,7 @@ if [ -f "${ZIPPED_REPO}" ]; then
   fi
 else
   # Download offliner runtime
-  curl -v -L http://repo.maven.apache.org/maven2/com/redhat/red/offliner/offliner/$OFFLINER_VERSION/offliner-$OFFLINER_VERSION.jar > /tmp/offliner.jar
+  curl -v -L https://repo.maven.apache.org/maven2/com/redhat/red/offliner/offliner/$OFFLINER_VERSION/offliner-$OFFLINER_VERSION.jar > /tmp/offliner.jar
```

Comment:
curl -o $TMPDIR/offliner.jar seems slightly better here?

Reply:
Thanks fixed, TMPDIR doesn't exist in centos7 image. Keeping /tmp

Reply:
Yeah, this is probably fine as /tmp is posix. I guess the safest version would be to use tmpdir="${TMPDIR:-/tmp}" for the case that someone wanted a different directory for this when they were building, but I bet we have other uses of /tmp around too, and I don't see an immediate use case.

Reply:
@luck3y , I logged https://github.com/wildfly/wildfly-cekit-modules/issues/149 to track your more general feedback.